scripts/linux/install-kubernetes.sh

```diff
@@ -13,16 +13,40 @@ while true; do
   esac
 done
 
+debug_output="$1"
+
+echo "debug_output: $debug_output"
+
+verbose_flag=
+if [ "$debug_output" = "on" ]; then
+    verbose_flag="-vv"
+fi
+
+echo "verbose_flag: $verbose_flag"
+
 echo "Ensure the Docker service is enabled and running"
+
 systemctl enable docker
 systemctl restart docker
 
 inventory="/etc/"$inventory
 
 echo "Running Ansible playbooks against $inventory inventory"
-docker run --rm \
-    -v /vagrant/ansible:/etc/ansible \
-    -v /vagrant/ansible/playbooks/files/tls:/opt/tls/self_signed \
-    --net=host \
-    ferrarimarco/open-development-environment-ansible:2.7.12-alpine \
-    /bin/sh -c "ansible-galaxy install -r /etc/ansible/requirements.yml && ansible-playbook -i $inventory /etc/ansible/playbooks/kubernetes.yml && ansible-playbook -i $inventory /etc/ansible/playbooks/openssl-self-signed-certificate.yml"
+
+if [ "$debug_output" = "on" ]; then
```

Comment:
Remove this `if` and leave output redirection on in any case.

Reply:
done